pkg/execution/state/redis_state/queue_processor.go

```diff
@@ -1133,8 +1137,8 @@ func (q *queue) capacity() int64 {
 // queue items.
 func (q *queue) peekSize() int64 {
 	f := q.capacity()
-	if f > QueuePeekMax {
-		return QueuePeekMax
+	if f > QueuePeekDefault {
+		return QueuePeekDefault
```

Comment:
maybe this should be an environment variable setting instead of a const.
and just use the default value if it's not set.

Reply:
Thinking that we we shouldn't downcast from max to default — if we do that, we should lower the max.  We could either lower the Default and the Max, or update the consts that we use when we initialize the queue in OSS and the cloud.